docs/mission_statement.md

```diff
@@ -0,0 +1,19 @@
+## [WebsiteOne](http://agileventures.org/)
+## Mission statement
+#### To provide a platform for online collaboration and crowdsourced project development.
```

Comment:
I'm thinking to replace with "To provide a coordination point for new members hoping to join existing teams, start new projects and connect with charitable causes"